armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -392,18 +441,18 @@ ELECTRS_VERSION="0.6.1"
 # ./install.sh
 #
 # apt install clang cmake
-# git clone https://github.com/romanz/electrs
+# git clone https://github.com/romanz/electrs || true
```

Comment:
Nit: Do we need to add the `|| true` to a commented-out command?

Actually, do we need to keep the commented-out commands in the script in the first place?

Reply:
I am planning to clean up the overall build (also remove unused packages at the end) in a seperate pull request.